### DISCUSSION TITLE
Several Miele Machines - Consumption values

### DISCUSSION THREAD

**Discussion by europeke** [astrandb/miele#455] [Q&A]
Hi!
I have running several Miele appliances at my home. All Miele@Home appliances. Washmachine, Dishwasher, Stove, Oven, Dry Tumbler.
I interessted to split up consumption values i can get.

I noted that in the official Miele App itself electrical consumtpion of washmaschine, dry tumbler, oven, stove is not present. Stove and oven as well not present with electrical consumption. Dishwasher has electrial and water consumption. All devices are latest firmware. Washmachine always shows 0,1kW consumption all the time in the app.

I saw the note that values are depending on availbilities of appliance. I guess therefore i am not able to get the values in Home Assistant as well. I saw they were hidden for example at the washmachine. However making them visible does not result in having consumption values. 

Is there any last chance to get them or do we need to wait until miele is pushing them? 

At the moment (for electrical and water):
Dishwasher - water, electrical
Washmachine - water
Dry Tumbler - none
Stove - none
Oven - none

Many thanks for support. If you need appliance specs or any data let me know.

**Comment by astrandb** [comment]
We collect the consumption data if it is pushed by Miele API. 
If it not there it is not there...
We chose to make the entities disabled by default since the support documentation is incomplete
and we got more questions about non-changing entities than about non-displayed ones.

**Comment by PortInnovator** [comment]
@europeke Have you asked Miele Service for support? When I noticed that the energy consumption of the washing machine in the app was gone after a few month I asked for support using the app and received a call by Miele Service one day later. There is a bug in the FW, switching off the energy consumption when the heater reports a single error (i.e. water temperature too low). The Miele technician reset the internal error counter of the washing machine and updated the FW (not officially available). Now the app is working correctly again and the Miele HA integration as well.
